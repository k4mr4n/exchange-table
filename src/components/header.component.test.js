import React from 'react'
import { render } from '@testing-library/react'

import HeaderComponent from './header.component'

test('Header Component', async () => {
  const { baseElement } = render(<HeaderComponent />)

  expect(baseElement).toMatchSnapshot()
})
