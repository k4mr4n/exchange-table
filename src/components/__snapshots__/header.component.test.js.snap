// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component 1`] = `
<body>
  <div>
    <header
      class="MuiPaper-root MuiPaper-elevation4 MuiAppBar-root MuiAppBar-positionRelative MuiAppBar-colorPrimary"
    >
      <div
        class="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
      >
        <span
          aria-hidden="true"
          class="material-icons MuiIcon-root makeStyles-icon-1"
        >
          trending_up
        </span>
        <h6
          class="MuiTypography-root MuiTypography-h6 MuiTypography-colorInherit MuiTypography-noWrap"
        >
          ExchangeRates
        </h6>
      </div>
    </header>
  </div>
</body>
`;
